profile/README.md

```diff
@@ -1,28 +1,47 @@
 # Veecle
 
-Veecle | All Cars. One Software. Our mission is to bridge the gap between existing supplier components and smart apps running on top through unified software. We provide a platform to tackle next-gen automotive connected & security demands. 
+Our mission is to bridge the gap between existing supplier components and smart apps running on top through unified software framework. We provide a platform to tackle next-gen automotive connected & security demands.
 
-Veecle has 3 different product lines: 
+### AURIX™ Rust Startup Ecosystem
 
-- **NoS**: a 100% Rust heap-free, asynchronous and type-safe runtime for automotive. NoS acts as the gateway that allows Nteract to communicate with the vehicle in a secure and safe way.  This unit is secure by design, updatable over-the-air, and fully testable over virtual pipelines. Designed for rapid and agile development 
+The AURIX™ Rust Startup Ecosystem is a collaborative effort involving [Veecle](https://www.veecle.io), [Infineon](https://www.infineon.com), [HighTec](https://hightec-rt.com) and [Bluewind](https://www.bluewind.it) aimed at supporting Rust on Infineon's AURIX™ architecture for automotive and industrial applications.
 
-- **Nteract:** a 100% Rust Android Automotive & Linux compatible HAL providing secure & safe APIs to communicate to any vehicle. Nteract acts as the security gateway between the non-secure, connected "Android & Linux" world and the in-vehicle highly secure domain. This allows for zero-cost secure integration of HMI and applications 
+Veecle contributes with two keys components to the ecosystem:
 
-- **Veecle Studio & Cloud Services** consist of a set of tools that can be used to deploy, test, monitor and manage our instances. At its core, this is a set of secure APIs that allows interconnecting visual tools for development, deployment, data visualisation, data storage and fleet management
+- [Veecle-pxros](https://github.com/veecle/veecle-pxros) A runtime for writing Rust applications on top of HighTec's safety kernel for the Infineon AURIX™ Platform.
+- [Tricore-probe](https://github.com/veecle/tricore-probe): A tool that facilitates the flashing and debugging of Rust code on AURIX™ chips.
 
-## What is currently published
+### Android Automotive support
 
-- [TC37x-demo](https://github.com/veecle/tc37x-demo): demo code to run a 100% Rust basic CAN driver on AURIX™ TC375 LK
-- [Tricore-probe](https://github.com/veecle/tricore-probe): a Rust probe for AURIX™ (currently works on Windows and Linux)  
+We support [Android Automotive](aaos): this enables customers to develop safe and efficient interfaces to the vehicle. 
```

Comment:
The link label is not being resolved if I view the file here:

https://github.com/veecle/.github/blob/b7831ef6e59043330de3d79e6e1b20a70b70b3ef/profile/README.md

I think this happens with the rest of links.